### PR TITLE
trivial: add some missing dashifications

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: doc/bips.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.18.0**):
+BIPs that are implemented by Dash Core (up-to-date up to **v18.0**):
```
Comment (PastaPastaPasta) [inline doc/bips.md:1]: I don't think this is updated. @thephez you want to make sure this is updated?

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

File: doc/bips.md (hunk 1 above)
Comment (thephez) [reply doc/bips.md:1]: See #4780

**Review by thephez — APPROVED** [review]
LGTM :+1:

**Comment by PastaPastaPasta** [comment]
same, milestone @UdjinM6

**Pull request merged**